tests/ImageSharp.Tests/Formats/Jpg/Utils/ReferenceImplementations.LLM_FloatingPoint_DCT.cs

```diff
@@ -168,27 +166,27 @@ internal static void iDCT2D_llm(Span<float> s, Span<float> d, Span<float> temp)
             /// </summary>
             /// <param name="s">Source</param>
             /// <param name="d">Destination</param>
-            public static void fDCT2D8x4_32f(Span<float> s, Span<float> d)
+            public static void FDCT2D8x4_32f(Span<float> s, Span<float> d)
             {
-                Vector4 c0 = _mm_load_ps(s, 0);
-                Vector4 c1 = _mm_load_ps(s, 56);
-                Vector4 t0 = (c0 + c1);
-                Vector4 t7 = (c0 - c1);
-
-                c1 = _mm_load_ps(s, 48);
-                c0 = _mm_load_ps(s, 8);
-                Vector4 t1 = (c0 + c1);
-                Vector4 t6 = (c0 - c1);
-
-                c1 = _mm_load_ps(s, 40);
-                c0 = _mm_load_ps(s, 16);
-                Vector4 t2 = (c0 + c1);
-                Vector4 t5 = (c0 - c1);
-
-                c0 = _mm_load_ps(s, 24);
-                c1 = _mm_load_ps(s, 32);
-                Vector4 t3 = (c0 + c1);
-                Vector4 t4 = (c0 - c1);
+                Vector4 c0 = Mm_load_ps(s, 0);
```

Comment:
Nope, stylecop, you failed terribly again!

This code has been kept deliberately similar [to the original](https://github.com/norishigefukushima/dct_simd/blob/master/dct/dct8x8_simd.cpp). and `_mm_load_ps` refers to the corresponding C++ intrinsic to emphasize compatibility with that implementation.

Reply:
Stylecop hasn't failed here. The rule is good, this is an edge case.

The fix was based on a lack of critical information.

I would add the exception inline with a justification describing why this is important.

Reply:
Agree, just trying to be dramatic 😄

The point is that we need to be careful with the automatic application of the rules. If something needs to be suppressed too often, then re-evaluate the rule. Fortunately this doesn't seem to be the case with the current set.

Reply:
Agree, the rules are not fixed and as soon as one causes significant issue we should definitely exclude it. 

All I'm looking for by adding rules is a little bit more structure to our tests. A lot of different people have submitted significant parts of code now and it's become, for me, at least difficult to follow at times.

The less draconian the ruleset the better. Let's just be careful to not [throw the baby out with the bathwater](https://en.wikipedia.org/wiki/Don%27t_throw_the_baby_out_with_the_bathwater).

Reply:
i have reverted it back, to be more like the original code again: 94dc54b4144394f125673066487c0e5a18036a04